docs/current_docs/cookbook/cookbook.mdx

```diff
@@ -1171,6 +1170,39 @@ Retrieve the value of the key `foo` after recreating the service state from the
 dagger call get --key=foo
 ```
 
+### Execute functions concurrently
+
+The following Dagger Function demonstrates how to use native-language concurrency features ([errgroups](https://pkg.go.dev/golang.org/x/sync/errgroup) in Go, [task groups](https://docs.python.org/3/library/asyncio-task.html) in Python), and [promises](https://basarat.gitbook.io/typescript/future-javascript/promise) in TypeScript to execute other Dagger Functions concurrently. If any of the concurrently-running functions fails, the remaining ones will be immediately cancelled.
```

Comment:
You didn't update the TaskGroup link as I suggested:
```suggestion
The following Dagger Function demonstrates how to use native-language concurrency features ([errgroups](https://pkg.go.dev/golang.org/x/sync/errgroup) in Go, [task groups](https://anyio.readthedocs.io/en/stable/tasks.html) in Python), and [promises](https://basarat.gitbook.io/typescript/future-javascript/promise) in TypeScript to execute other Dagger Functions concurrently. If any of the concurrently-running functions fails, the remaining ones will be immediately cancelled.
```

Reply:
Ah, sorry, I think I misunderstood this because it said AnyIO in both places in your original comment: `The AnyIO TaskGroup is [not the same](https://anyio.readthedocs.io/en/stable/tasks.html#differences-with-asyncio-taskgroup) as the AnyIO TaskGroup: https://anyio.readthedocs.io/en/stable/tasks.html`

Thanks for catching this!

Reply:
Oh, sorry about that! 😅